packages/react-tabs/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@fluentui/react-tabs",
+  "version": "9.0.0-beta.3",
+  "private": true,
+  "description": "Tabs allow for navigation between two or more content views and relies on text headers to articulate the different sections of content.",
```

Comment:
I think this should be a bit more general--typically we use something like "Tabs component for Fluent UI React" (this is the description shown in the npm registry)

Reply:
I pulled this from the figma description for the component. I can change it to something more title-like as I get the started on the components.